subjects/multorsum/README.md

```diff
@@ -0,0 +1,49 @@
+## multorsum
+
+### Instructions
+
+Write a function that receives a slice of `int` and an `int` representing the initial value as arguments. 
+
+You should traverse the slice and for each `int` check the following restrictions:
```

Comment:
I would suggest to rephrase this  as something like: "The function should go through the slice......restrictions:"

Reply:
Agreed. Fixed